src/lang/Messages.properties

```diff
@@ -841,6 +841,7 @@ conn.options.proxy.excluded.domain.dialog.remove.button.confirm = Remove
 conn.options.proxy.excluded.domain.dialog.remove.checkbox.label = Do not show this message again
 conn.options.proxy.useProxyChain = Use proxy chain
 conn.options.singleCookieRequestHeader = Single Cookie Request Header
+conn.options.httpStateEnabled = Enable (Global) HTTP state
```

Comment:
Please add a leading cap to `state` so that it's in title caps.
https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#text-titles-menu-items-options-and-tool-tips